backend/matching-service/src/services/rabbitmq.service.ts

```diff
@@ -262,6 +276,42 @@ class RabbitMQConnection {
     createId(userId: string, timestamp: string): string {
         return userId + timestamp
     }
+
+    async listenToDeadLetterQueue() {
+        try {
+            if (!this.channel) {
+                await this.connect()
+            }
+
+            // DeadLetter-Queue setup parameters
+            const DLX_QUEUE = 'deadletter-queue'
+            const DLX_EXCHANGE = 'dlx'
+            const DLX_ROUTING_KEY = 'dlx-key'
+            await this.channel.assertExchange(DLX_EXCHANGE, 'direct', { durable: true })
```

Comment:
Is there a reason why durable is used for the DLQ?

Reply:
Sure, we can remove durability as long as the WS has an automatic timeout and disconnects. Otherwise durability will be important else WS connections of lost DL messages may remain open for very long unintentionally. In this case durability and message persistence (then need TODO) will be necessary, but I believe for our use case we can just make WS close automatically.